Sources/MockoloFramework/Parsers/SwiftSyntaxExtensions.swift

```diff
@@ -33,16 +33,16 @@ extension SyntaxProtocol {
 
 extension AttributeListSyntax {
     var trimmedDescription: String? {
-        return self.withoutTrivia().description.trimmingCharacters(in: .whitespacesAndNewlines)
+        return self.trimmed.description.trimmingCharacters(in: .whitespacesAndNewlines)
     }
 }
 
-extension ModifierListSyntax {
+extension DeclModifierListSyntax {
     var acl: String {
         for modifier in self {
             for token in modifier.tokens(viewMode: .sourceAccurate) {
                 switch token.tokenKind {
-                case .publicKeyword, .internalKeyword, .privateKeyword, .fileprivateKeyword:
+                case .keyword(.public), .keyword(.internal), .keyword(.private), .keyword(.fileprivate):
```

Comment:
- 5.9 Add new `package` keyword
- No need to treat `open` specially anymore.

```swift
                case .keyword(.public), 
                        .keyword(.internal),
                        .keyword(.private),
                        .keyword(.fileprivate),
                        .keyword(.package),
                        .keyword(.open):
                    return token.text
                default:
                    return ""
```

Reply:
Thanks for catching that, I've pushed 519439fb550dc23e622897de0080ce8ba68ddd78 to handle these.